### PR TITLE
check file is not a directory when returned as a file

### PR DESCRIPTION
**Pull request by rickyshrestha**
fix for #203

### CODE REVIEW

File: v2/jam/parser/finder.go [hunk 1]
```diff
@@ -3,14 +3,15 @@ package parser
...
 	"path/filepath"
 	"strings"
 	"time"
 
 	"github.com/gobuffalo/packr/v2/plog"
 	"github.com/karrick/godirwalk"
-	"github.com/gobuffalo/packr/v2/internal/takeon/github.com/markbates/errx"
-	"github.com/gobuffalo/packr/v2/internal/takeon/github.com/markbates/oncer"
+	"github.com/markbates/oncer"
+	"github.com/pkg/errors"
```
Comment (markbates) [inline v2/jam/parser/finder.go]: Th pkg/errors package has recently been removed from packr, can you please remove its use?

File: v2/jam/parser/finder.go [hunk 2]
```diff
@@ -3,14 +3,15 @@ package parser
...
+	"os"
 	"path/filepath"
 	"strings"
 	"time"
 
 	"github.com/gobuffalo/packr/v2/plog"
 	"github.com/karrick/godirwalk"
-	"github.com/gobuffalo/packr/v2/internal/takeon/github.com/markbates/errx"
-	"github.com/gobuffalo/packr/v2/internal/takeon/github.com/markbates/oncer"
+	"github.com/markbates/oncer"
```
Comment (markbates) [inline v2/jam/parser/finder.go]: Can you please revert back to using the internal packages and not their external paths?

File: v2/jam/parser/finder.go [hunk 3]
```diff
@@ -52,7 +64,7 @@ func (fd *finder) findAllGoFilesImports(dir string) ([]string, error) {
 		ctx := build.Default
 
 		if len(ctx.SrcDirs()) == 0 {
-			err = fmt.Errorf("no src directories found")
+			err = errors.New("no src directories found")
```
Comment (markbates) [inline v2/jam/parser/finder.go]: Please revert this line

File: v2/jam/parser/finder.go [hunk 4]
```diff
@@ -63,7 +75,7 @@ func (fd *finder) findAllGoFilesImports(dir string) ([]string, error) {
 
 		if err != nil {
 			if !strings.Contains(err.Error(), "cannot find package") {
-				if _, ok := errx.Unwrap(err).(*build.NoGoError); !ok {
```
Comment (markbates) [inline v2/jam/parser/finder.go]: Please revert this line

**Comment by rickyshrestha** [comment]
changes are now made on the latest version.

**Comment by markbates** [comment]
Thanks! I’ll do a release shortly.

**Pull request merged**